src/graphql/schema.gql

```diff
@@ -391,6 +391,7 @@ type Space {
   verified: Boolean
   flagged: Boolean
   hibernated: Boolean
+  turbo: Boolean
```

Comment:
I feel like this should be a number, we may add different levels of turbo in the future, like for example higher pricing plan or just a spam-fighting one